### PR TITLE
#339 Add schema selector for concatenation rule

### PR DESCRIPTION
**Pull request by GeorgiChochov**
* Fixed broken concatenation output nullability check
* Added Schema Field Selector

### CODE REVIEW

**Comment by GeorgiChochov** [comment]
Thanks, JS refactoring can sometimes go a little funny (unintentional method rename)

**Comment by GeorgiChochov** [comment]
> Added 4th rule and got Harlem Shake
> 
> `Uncaught TypeError: Cannot read property 'collapseAll' of undefined`
> 
> In general, whenever I add another field it does this. Sometimes 3 times sometimes doesn't stop.

That's wild. I can't reproduce it...

**Comment by Zejnilovic** [comment]
The Harlem Shake happens when the line of the concatenation field is too long it seems.

**Comment by GeorgiChochov** [comment]
I'll finish up #82 before getting back to this.

File: menas/ui/index.html [hunk 1]
```diff
@@ -38,6 +38,8 @@
     <script src="components/dataset/conformanceRule/schemaManager.js"></script>
     <script src="components/dataset/conformanceRule/MappingConformanceRule/JoinConditionDialog.js"></script>
     <script src="components/dataset/conformanceRule/MappingConformanceRule/JoinConditionDialogFactory.js"></script>
+    <script src="components/dataset/conformanceRule/ConcatenationConformanceRule/ConcatenationColumnDialog.js"></script>
+    <script src="components/dataset/conformanceRule/ConcatenationConformanceRule/ConcatenationColumnDialogFactory.js"></script>
```
Comment (lokm01) [inline menas/ui/index.html]: I wish ES6 imports worked without these... 🙄

Comment (GeorgiChochov) [reply menas/ui/index.html]: They do if you use a real bundler, but UI5 doesn't play ball with ES6 or any transpiling bundler (#314). That's something I'll figure out how to make work in the future. Already wasted 2 todays on it a while ago.

**Comment by GeorgiChochov** [comment]
I really can't reproduce the Harlem Shake... But I fixed the preselect error.

**Comment by Zejnilovic** [comment]
Can't reproduce on new commits either.

**Pull request merged**